docs/src/gs/install-cluster-api.md

```diff
@@ -23,25 +23,46 @@
    ```
 
 ## Configure authentication
+Before installing Cluster API Provider for OCI (CAPOCI), you must first set up your preferred 
+authentication mechanism using specific environment variables.
 
-Before installing Cluster API Provider for OCI (CAPOCI), you must first set up your preferred authentication mechanism using specific environment variables:
+### User Principal
+If the management cluster is hosted outside OCI, for example a Kind cluster, please configure
+user principal using the following parameters. Please refer to the [doc][api-signing-key] to generate the required
+credentials.
 
    ```bash
-      export OCI_TENANCY_ID=<tenancy-id>
-      export OCI_USER_ID=<user-id>
-      export OCI_CREDENTIALS_FINGERPRINT=<fingerprint>
-      export OCI_REGION=<region>
+      export OCI_TENANCY_ID=<insert-tenancy-id-here>
+      export OCI_USER_ID=<inser-user-ocid-here>
+      export OCI_CREDENTIALS_FINGERPRINT=<insert-fingerprint-here>
+      export OCI_REGION=<insert-region-here>
       # if Passphrase is present
-      export OCI_CREDENTIALS_PASSPHRASE=<passphrase>
+      export OCI_CREDENTIALS_PASSPHRASE=<insert-passphrase-here>
       export OCI_TENANCY_ID_B64="$(echo -n "$OCI_TENANCY_ID" | base64 | tr -d '\n')"
       export OCI_CREDENTIALS_FINGERPRINT_B64="$(echo -n "$OCI_CREDENTIALS_FINGERPRINT" | base64 | tr -d '\n')"
       export OCI_USER_ID_B64="$(echo -n "$OCI_USER_ID" | base64 | tr -d '\n')"
       export OCI_REGION_B64="$(echo -n "$OCI_REGION" | base64 | tr -d '\n')"
-      export OCI_CREDENTIALS_KEY_B64=$(base64 < <path-to-api-private-key-file> | tr -d '\n')
+      export OCI_CREDENTIALS_KEY_B64=$(base64 < <insert-path-to-api-private-key-file-here> | tr -d '\n')
       # if Passphrase is present
       export OCI_CREDENTIALS_PASSPHRASE_B64="$(echo -n "$OCI_CREDENTIALS_PASSPHRASE" | base64 | tr -d '\n')"
    ```
 
+### Instance Principal
```

Comment:
Should we call out somewhere that if the user sets the Instance Principal to `true` that will take precedence over User Principals?

Reply:
added a statement for this, please check.